### ISSUE TITLE
undo stacking during startup

### ISSUE THREAD

**Issue by matkoniecz** [westnordost/StreetComplete#1480]
Step 1: solve two different quest

Step 2: Switch applications, use system killer of apps

Step 3: Start StreetComplete, as it starts keep clicking "Undo" button

As result multiple undo windows will appear, each allowing to keep undoing. It is possible to undo more times than solved quests and get negative star count.

**Comment by westnordost** [comment]
You mean the "Undo" Button was clicked several times before the dialog popped up?

**Comment by matkoniecz** [comment]
> You mean the "Undo" Button was clicked several times before the dialog popped up?

Yes, and in case of opening undo dialog before killing app - old undo dialog is still present at the bottom, but during startup there is a moment where it is not visible and it is still possible to click undo button multiple time.

I suspect that it is easier to reproduce/trigger on phones with lower performance, even if that is not a phone specific bug.
